auto_nag/scripts/regression.py

```diff
@@ -163,11 +99,21 @@ def history_handler(history, data):
         return bugids - data
 
     def get_bugs(self, date='today', bug_ids=[]):
+        # Retrieve bugs to analyze.
         bugids = super(Regression, self).get_bugs(date=date, bug_ids=bug_ids)
-        data = self.analyze_comments(bugids['others'])
-        reg_bugids = {bugid for bugid, reg in data.items() if reg}
+
+        # Analyze bugs.
+        classes = self.model.classify(bugids['others'])
```

Comment:
bugbug is going to download the data automatically if we pass the IDs, so here it would be better to pass the bug objects with comments and history, so we don't have to download them twice (once in bugbug and once a few lines after here).

Reply:
Done